src/dashboard/apps/consent/views.py

```diff
@@ -108,7 +108,11 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context["control_authority"] = settings.CONSENT_CONTROL_AUTHORITY
         context["entity"] = self._get_entity()
-        context["consents"] = self._get_entity().get_consents()
+        context["consents"] = (
+            self._get_entity()
+            .get_consents()
+            .order_by("delivery_point__station_name", "provider_assigned_id")
```

Comment:
Why not adding default ordering to the `Consent` model instead?

Reply:
probably because I didn't think about it  :sweat_smile: